packages/core/src/utils/rateLimiting/userBasedRateLimiter.ts

```diff
@@ -104,3 +113,10 @@ function userHasOakEmail(user: User) {
       !email.emailAddress.includes("demo"),
```

Comment:
Doesn't this mean that if I have `someone+demo@example.com` I will bypass the rate limiting?